nautobot/dcim/templates/dcim/interfaceredundancygroupassociation_create.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+
+{% block form %}
+<div class="panel panel-default">
+    <div class="panel-heading"><strong>Interface Redundancy Group</strong> {{ object }}</div>
+    <div class="panel-body">
+        {% render_field form.interface_redundancy_group %}
+        {% render_field form.region %}
+        {% render_field form.site %}
+        {% render_field form.rack %}
+        {% render_field form.device %}
+        {% render_field form.interface %}
+        {% render_field form.priority %}
```

Comment:
Double check me on this but I wonder if you can replace these lines with `{% render_form form %}`

Reply:
updated code